### ISSUE TITLE
High Levels of Loss on Training Model

### ISSUE THREAD

**Issue by tinybighuman** [niemtec/Effects-of-Image-Augmentation-on-CNNs#4]
Models produce a high level of loss ranging from 8.0 to 0.8.

[Stack Overflow - Reducing Loss Between Epochs](https://stackoverflow.com/questions/35599328/keras-unable-to-reduce-loss-between-epochs)

[Using Optimisers in Keras](https://keras.io/optimizers/)
